src/Nox.Lib/Presentation/Api/OData/ODataApi.cs

```diff
@@ -12,5 +14,34 @@ public static string GetRoutePrefix(string apiRoutePrefix)
         {
             return apiRoutePrefix.TrimStart('/');
         }
+
+        public static Dictionary<string, dynamic> GetDeltaUpdatedProperties<T>(Delta<T> delta) where T : class
+        {
+            var updatedProperties = new Dictionary<string, dynamic>();
+
+            foreach (var propertyName in delta.GetChangedPropertyNames())
+            {
+                if (delta.TryGetPropertyValue(propertyName, out dynamic value))
+                {
+                    var property = typeof(T).GetProperty(propertyName);
+                    if (property is null)
+                        continue;
+
+                    var propertyType = property.PropertyType;
+                    if (propertyType.IsClass && propertyType != typeof(string) && value != null)
```

Comment:
```suggestion
                    if (typeof(propertyType).GetInterfaces().Contains(typeof(INoxCoumpoundType)) && value != null)
```
It can be added INoxCompoundType to identify all Nox Compound types. What do you think @rochar?